content/es/docs/concepts/storage/storage-capacity.md

```diff
@@ -0,0 +1,75 @@
+
+## Planificación
+
+El Planificador de Kubernetes utiliza la información sobre la capacidad de almacenamiento si:
+
+- la Feature Gate de `CSIStorageCapacity` es `true`,
+- un Pod usa un volumen que aún no se ha creado,
+- ese volumen usa un {{< glossary_tooltip text="StorageClass" term_id="storage-class" >}} que hace referencia a un controlador CSI y usa el [modo de enlace de volumen](/docs/concepts/storage/storage-classes/#volume-binding-mode) `WaitForFirstConsumer` ,
+  y
+- el objeto `CSIDriver` para el controlador tiene `StorageCapacity` establecido en `true`.
+
+En ese caso, el Planificador sólo considera los nodos para el Pod que tienen suficiente almacenamiento disponible. Esta verificación es muy simplista y solo compara el tamaño del volumen con la capacidad indicada en los objetos `CSIStorageCapacity` con una topología que incluye el nodo.
+
+Para los volúmenes con el modo de enlace de volumen `Immediate`, el controlador de almacenamiento decide dónde crear el volumen, independientemente de los pods que usarán el volumen.
+Luego, el Planificador programa los pods en los nodos donde el volumen está disponible después de que se haya creado.
+
+Para los [volúmenes efímeros de CSI](/docs/concepts/storage/volumes/#csi),
+la programación siempre ocurre sin considerar la capacidad de almacenamiento. Esto se basa en la suposición de que este tipo de volumen solo lo utilizan controladores CSI especiales que son locales a un nodo y no necesitan allí recursos importantes.
+
+## Reprogramar
+
+Cuando se selecciona un nodo para un Pod con volúmenes `WaitForFirstConsumer`, esa decisión sigue siendo tentativa. El siguiente paso es que se le pide al controlador de almacenamiento CSI que cree el volumen con una pista de que el volumen está disponible en el nodo seleccionado.
+
+Debido a que Kubernetes pudo haber elegido un nodo basándose en información de capacidad desactualizada, es posible que el volumen no se pueda crear realmente. Luego, la selección de nodo se restablece y el Planificador de Kubernetes intenta nuevamente encontrar un nodo para el Pod.
+
+## Limitaciones
+
+El seguimiento de la capacidad de almacenamiento aumenta las posibilidades de que la programación funcione en el primer intento, pero no puede garantizarlo porque el planificador tiene que decidir basándose en información potencialmente desactualizada. Por lo general, el mismo mecanismo de reintento que para la programación sin información de capacidad de almacenamiento es manejado por los errores de programación.
+
+Una situación en la que la programación puede fallar de forma permanente es cuando un Pod usa varios volúmenes: es posible que un volumen ya se haya creado en un segmento de topología que luego no tenga suficiente capacidad para otro volumen. La intervención manual es necesaria para recuperarse de esto, por ejemplo, aumentando la capacidad o eliminando el volumen que ya se creó. [
```

Comment:
Una situación en la que la planificación puede fallar de forma permanente es cuando un Pod usa varios volúmenes: es posible que un volumen ya se haya creado en un segmento de topología que luego no tenga suficiente capacidad para otro volumen. La intervención manual es necesaria para recuperarse de esto, por ejemplo, aumentando la capacidad o eliminando el volumen que ya se creó. [

Reply:
Hola @gamba47 puedes resaltar dónde exactamente es la propuesta de cambio? Que palabra o frase. :)

Reply:
@gamba47 algunos comentarios es mas facil de comprenderlos a traves de las sugerencias, checa [el punto 6](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request)

Reply:
Hola a ambos, los reviso nuevamente para ser mas preciso, lo puse asi porque cambie una frase mas que palabras, pero si no se me entiende lo cambio, les pido perdon por los inconvenientes, ya voy a estar a la altura!

Reply:
Hola Edith, el cambio era en la linea 63, en vez de programación me paerce que va mejor "planificación", la frase en ingles es:
`One situation where scheduling can fail permanently is when a Pod...`
Pero en castellano no se que suena mejor, si planificar enviar un pod o programar enviar un pod, solo por eso hice el comentario.
Saludos!

Reply:
Buen punto @gamba47 , tampoco lo sé, vamos a preguntarlo en la reunión del martes :)
Gracias por tu feedback.

Reply:
Me decanto por utilizar "_planificar_" cómo traducción de "**schedule**", tienen un sentido menos ambiguo. Usemos "_programar_" para "**programming**"